### ISSUE TITLE
Lack of tempo in score does not gracefully exit [Regression]

### ISSUE THREAD

**Issue by hyperdriveguy** [nephitejnf/muse2pokecrystal#21]
Traceback:
```
muse2pokecrystal/src/score.py", line 66, in process_to_file_store
    bpm = int(channel_part.find(
AttributeError: 'NoneType' object has no attribute 'get'
```

**Comment by hyperdriveguy** [comment]
Fixed in #20

**Comment by hyperdriveguy** [comment]
Fixed in #20 (For real now)
